runtime/src/bank.rs

```diff
@@ -2615,6 +2622,49 @@ impl Bank {
         }
     }
 
+    pub fn calculate_capitalization(&self) -> u64 {
+        self.get_program_accounts(None)
+            .into_iter()
+            .filter_map(|(_pubkey, account)| {
+                if account.lamports == u64::max_value() {
```

Comment:
A brief comment about these pool accounts would be nice too.

Reply:
I removed this as this handling is no longer needed.

I originally added this exception according to this: https://github.com/solana-labs/solana/pull/9936/files#r422076284.
But it seems that this is no longer needed according to this: https://github.com/solana-labs/solana/pull/9992/files#r457126438